### DISCUSSION TITLE
AttributeError: 'Config' object has no attribute 'logging_format'

### DISCUSSION THREAD

**Discussion by Arwin-CTM** [hrcorval/behavex#105] [Q&A] [answered]
i installed behavex and ran the command -> behavex -t @testtag --parallel-processes 4 --parallel-scheme scenario.

Getting the below error. Can i please know how to fix this?

AttributeError: 'Config' object has no attribute 'logging_format'

**Comment by databene** [comment]
Same error here, Did you find a solution?

> 18:20:28    File "/usr/local/lib/python3.8/site-packages/behavex/runner.py", line 511, in _launch_behave
18:20:28      execution_code = behave_script.main(behave_args)
18:20:28    File "/usr/local/lib/python3.8/site-packages/behave/__main__.py", line 183, in main
18:20:28      return run_behave(config)
18:20:28    File "/usr/local/lib/python3.8/site-packages/behave/__main__.py", line 127, in run_behave
18:20:28      failed = runner.run()
18:20:28    File "/usr/local/lib/python3.8/site-packages/behave/runner.py", line 804, in run
18:20:28      return self.run_with_paths()
18:20:28    File "/usr/local/lib/python3.8/site-packages/behave/runner.py", line 824, in run_with_paths
18:20:28      return self.run_model()
18:20:28    File "/usr/local/lib/python3.8/site-packages/behave/runner.py", line 626, in run_model
18:20:28      failed = feature.run(self)
18:20:28    File "/usr/local/lib/python3.8/site-packages/behave/model.py", line 321, in run
18:20:28      failed = scenario.run(runner)
18:20:28    File "/usr/local/lib/python3.8/site-packages/behave/model.py", line 1114, in run
18:20:28      failed = scenario.run(runner)
18:20:28    File "/usr/local/lib/python3.8/site-packages/behave/model.py", line 758, in run
18:20:28      runner.run_hook("after_scenario", runner.context, self)
18:20:28    File "/usr/local/lib/python3.8/site-packages/behavex/environment.py", line 68, in run_hook
18:20:28      behavex_env.after_scenario(context, *args)
18:20:28    File "/usr/local/lib/python3.8/site-packages/behave/log_capture.py", line 218, in f
18:20:28      h = LoggingCapture(context.config, level=level)
18:20:28    File "/usr/local/lib/python3.8/site-packages/behave/log_capture.py", line 68, in __init__
18:20:28      if config.logging_format:
18:20:28  AttributeError: 'Properties' object has no attribute 'logging_format'

**Comment by hrcorval** [comment, marked as answer]
We have found an issue in the order in which the behavex hooks are being executed, and we realize this might have been the reason for the issue you are experiencing. 
Please, keep us posted if this change solved the issue. Otherwise, we will continue investigating on the way to reproduce it.
Thanks!
